examples/snn/Makefile

```diff
@@ -13,6 +13,22 @@ include ../BaseMakefile.mk
 ALGO = snn
 EXEC = bin/test_$(ALGO)_$(NVCC_VERSION)_$(ARCH_SUFFIX)
 
+FAISS_INC =
+FAISS_LINK =
+
+ifdef FAISS_INCLUDE_DIR
```

Comment:
Where is `FAISS_INCLUDE_DIR`? CMakeLists =/= BaseMakefile.mk, they don't see each other's variables. I think here you have to do something like what we do for `use_metis=1`. Check BaseMakefile.mk for example.

Reply:
FAISS_INCLUDE_DIR is environment variable. It has to be defined from console before compilation

Reply:
Can you add this comment to the Makefile? Just so we know for future?

Reply:
It is already there. While compilation it says: "FAISS not found. In order to use FAISS, please define FAISS_INCLUDE_DIR"